packages/website/components/mdx/code/inline.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { AkCode } from '@atlaskit/code';
+
+interface Props {
+  children: string;
+}
```

Comment:
Should inline examples not also have language support?

Reply:
Unfortunately there is no way to specify a language in the markdown for an inline component :(

Reply:
Sad truths be sad.